JSONAPI.TodoMVC.API/Controllers/TodosController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using JSONAPI.Core;
+using JSONAPI.EntityFramework;
+using JSONAPI.EntityFramework.Http;
+using JSONAPI.TodoMVC.API.Models;
+
+namespace JSONAPI.TodoMVC.API.Controllers
+{
+    public class TodosController : ApiController<Todo, TodoMvcContext>
+    {
+        protected readonly TodoMvcContext DbContext;
+
+        public TodosController(TodoMvcContext dbContext)
+        {
+            DbContext = dbContext;
+        }
+
+        protected override IMaterializer MaterializerFactory()
+        {
+            return new EntityFrameworkMaterializer(DbContext);
+        }
+    }
```

Comment:
Yeah, agreed. This is what I had to do in order to make the class testable. I think `IMaterializer` should become a constructor parameter on `ApiController`. Then we register an EntityFrameworkMaterializer as IMaterializer using Autofac.